changelogs/fragments/260-zabbix_user.yml

```diff
@@ -0,0 +1,4 @@
+bugfixes:
+  - zabbix_user - fixed it couldn't create a user since Zabbix 5.2 (https://github.com/ansible-collections/community.zabbix/pull/260).
```

Comment:
```suggestion
  - zabbix_user - fixed issue where module couldn't create a user since Zabbix 5.2 (https://github.com/ansible-collections/community.zabbix/pull/260).
```